doc/index.rst

```diff
@@ -1,51 +1,132 @@
-.. py:currentmodule:: lsst.ts.IntegrationTests
+..
```

Comment:
More the boiler plate comments here since they are distracting.

Reply:
fixed